crates/common/src/lib.rs

```diff
@@ -1,4 +1,4 @@
-#![cfg_attr(feature = "strict", deny(warnings))]
+// #![cfg_attr(feature = "strict", deny(warnings))]
```

Comment:
Do we want to leave this here as a comment?

Reply:
Nice catch! Thanks! Fixed.